R/hydrate.R

```diff
@@ -1,41 +1,24 @@
 
-#' Hydrate a Project
+#' Copy packages from user libraries to a project library
 #'
-#' Discover the \R packages used within a project, and then install those
-#' packages into the active library. This effectively allows you to fork the
-#' state of your default \R libraries for use within a project library.
+#' @description
+#' Copy/link the packages currently used by the project from a user library
+#' to the project library. This synchronises a project library with you have
+#' installed globally.
 #'
-#' It may occasionally be useful to use `renv::hydrate()` to update the packages
-#' used within a project that has already been initialized. However, be aware
-#' that it's possible that the packages pulled in may not actually be compatible
-#' with the packages already installed in the project library, so you should
-#' exercise caution when doing so.
+#' You generally don't need to call `hydrate()` directly as [init()] will call
+#' it for you. However, you may want to call it directly if you want to update
```

Comment:
I'd go a bit further and say that this isn't recommended or is experts-only, given the caveats of mixing different packages from different libraries.